JavaScript/sign-script.js

```diff
@@ -0,0 +1,160 @@
+    emailInput.classList.add('error'); // 에러 클래스 추가
+    return;
+  }
+  
+  if (emailInput.value !== '') {
+    errorMail.textContent = '이메일을 확인해주세요.';
+    emailInput.classList.add('error'); // 에러 클래스 추가
+    return;
+  } 
+  
+  if (emailInput.value === TEST_CORRECT_EMAIL && passInput.value === TEST_CORRENT_PASSWORD ) {
+    location.href = VALID_LINK;
+    return;
+  }
+
+  // errorMail.textContent = '';
+  // emailInput.classList.remove('error'); // 에러 클래스 제거
+}
+
+function checkPass() {
+  const passwordValue = passInput.value;
+  const hasMinLength = passwordValue.length >= 8;
+  const hasLetterAndNumber = /[a-zA-Z]/.test(passwordValue) && /\d/.test(passwordValue);
+
+  if (passInput.value === '') {
+    errorPass.textContent = '비밀번호를 입력해주세요.';
+    passInput.classList.add('error'); // 에러 클래스 추가
+    return;
+  }
+
+  if (!hasMinLength || !hasLetterAndNumber) {
+    errorPass.textContent = '비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.';
+    return;
+  }
+
+  if (passInput.value !== '') {
+    errorPass.textContent = '비밀번호를 확인해주세요.';
+    passInput.classList.add('error'); // 에러 클래스 추가
+    return;
+  }
+}
+
+function checkPassConfirm() {
+  if (passConfirmInput.value === '') {
+    errorPassConfirm.textContent = '비밀번호를 다시 입력해주세요.';
+    passConfirmInput.classList.add('error'); // 에러 클래스 추가
+  } else if (passConfirmInput.value !== passInput.value) {
+    errorPassConfirm.textContent = '비밀번호가 일치하지 않습니다.';
+    passConfirmInput.classList.add('error'); // 에러 클래스 추가
+  } else {
+    errorPassConfirm.textContent = '';
+    passConfirmInput.classList.remove('error'); // 에러 클래스 제거
+  }
+}
+
+
+function togglePasswordVisibility() {
+  const currentType = passInput.type;
+  passInput.type = (currentType === 'password') ? 'text' : 'password';
+  eyeImagePass.src = (currentType === 'password') ? 'img/eye-cross.png' : 'img/eye.png';
+}
+
+function togglePasswordVisibilityConfirm() {
+  const currentType = passConfirmInput.type;
+  passConfirmInput.type = (currentType === 'password') ? 'text' : 'password';
+  eyeImageConfirm.src = (currentType === 'password') ? 'img/eye-cross.png' : 'img/eye.png';
+}
+
+
+emailInput.addEventListener('focusout', checkEmail);
+passInput.addEventListener('focusout', checkPass);
+passConfirmInput.addEventListener('focusout', checkPassConfirm);
+eyeImagePass.addEventListener('click', togglePasswordVisibility);
+eyeImageConfirm.addEventListener('click', togglePasswordVisibilityConfirm);
+
+const TEST_SIGN_IN = {
+  "email" : "test@codeit.com",
+  "password": "sprint101",
+}
+
+fetch("https://bootcamp-api.codeit.kr/docs/api/sign-in", {
+  method : 'POST', 
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: JSON.stringify(TEST_SIGN_IN)
+})
+.then(data => {
+  console.log(data);
+
+  if (data.success) {
+    window.location.href = '/folder';
+  }
+});
+
+const TEST_CHECK_EMAIL = {
+  "email" : "test@codeit.com"
+}
+
+fetch("https://bootcamp-api.codeit.kr/docs/api/check-email", {
```

Comment:
```suggestion
const API_BASE_URL = 'https://bootcamp-api.codeit.kr';

fetch(`${API_BASE_URL }/docs/api/check-email`, {
```

[중요도: 중간] API경우 계속 사용할 수 있으니 기본 base url을 상수화하는 것을 추천드립니다.